PDP11/pdp11_rr.c

```diff
@@ -488,7 +492,7 @@ DEVICE rr_dev = {
 
 /* I/O dispatch routine, I/O addresses 17776710 - 17776736
 
-   17776710     RPDS    read-only except for attention bits
+   17776710     RPDS    read-only except for attention bits (write-clear)

```

Comment:
Was that the only thing that prevented the patch to be applied?

Reply:
No, it's something I noticed while reading the patch (which I haven't finished yet).  And the issue isn't the clarification but rather that the RPDS write handler seems to clear attention for any write, rather than clearing only attention bits corresponding to 1 bits in the data.

And yes, BIC RPDS,RPDS is obviously wrong; MOV RPDS,RPDS is the common and correct idiom for "clear all the write-1-to-clear bits"

Reply:
>  And the issue isn't the clarification but rather that the RPDS write handler seems to clear attention for any write,

I don't understand where you see that.

The relevant code is this (line 614):
```
           rpds &= ~(data & RPDS_ATTN);                /* clr attention bits */
```
which does the bit clear.

I am submitting this patch after XXDP'ing and DEC/X11'ing the device implementation, so if this basic thing wasn't handled correctly, the tests would not have passed.

Reply:
You're right, I misread it.  Thanks.